scripts/test_travis.sh

```diff
@@ -33,6 +33,7 @@ case "$1" in
     ;;
 
     core)
+        ./scripts/test_container.sh exm
```

Comment:
I would have said that kernel comes first and after the modules that depend on it :wink:

Reply:
Actually it's the other way around, kernel depends on exm

Reply:
:open_mouth: